### PR TITLE
Get vlan interfaces compliance test

### PR DESCRIPTION
**Pull request by emmurd**

### CODE REVIEW

File: tests/adapters/compliance_tests/get_vlan_interfaces_test.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright 2015 Internap.
```
Comment (JoProvost) [inline tests/adapters/compliance_tests/get_vlan_interfaces_test.py]: 2016!

**Comment by idjaw** [comment]
:+1:

File: tests/adapters/compliance_tests/add_vlan_test.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 Internap.
+# Copyright 2016 Internap.
```
Comment (JoProvost) [inline tests/adapters/compliance_tests/add_vlan_test.py]: Is this file really created in 2016?

File: tests/adapters/compliance_tests/get_interfaces_test.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 Internap.
+# Copyright 2016 Internap.
```
Comment (JoProvost) [inline tests/adapters/compliance_tests/get_interfaces_test.py]: Is this file really created in 2016?

File: tests/adapters/compliance_tests/get_vlan_test.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 Internap.
+# Copyright 2016 Internap.
```
Comment (JoProvost) [inline tests/adapters/compliance_tests/get_vlan_test.py]: Is this file really created in 2016?

File: tests/adapters/compliance_tests/set_vlan_icmp_redirects_state_test.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 Internap.
+# Copyright 2016 Internap.
```
Comment (JoProvost) [inline tests/adapters/compliance_tests/set_vlan_icmp_redirects_state_test.py]: Is this file really created in 2016?

File: tests/adapters/compliance_tests/get_interfaces_test.py (hunk 3 above)
Comment (idjaw) [inline tests/adapters/compliance_tests/get_interfaces_test.py]: If they weren't created in 2016, should be changed to 2015-2016

**Comment by emmurd** [comment]
I should have asked. I thought we had to update the copyright to the new year. These haven't been modified yet

**Comment by JoProvost** [comment]
Those files were created in 2015 and were not modified in 2016
- tests/adapters/compliance_tests/add_vlan_test.py
- tests/adapters/compliance_tests/add_vlan_test.py

But the other were created in January 2016 and should be updated, my be not in this PR

**Comment by JoProvost** [comment]
Could you please squash all your changes to avoid confusion in the master branch.

**Comment by JoProvost** [comment]
:+1:

**Pull request merged**